go/stellar/stellarsvc/frontend.go

```diff
@@ -993,6 +993,7 @@ func (s *Server) BuildPaymentLocal(ctx context.Context, arg stellar1.BuildPaymen
 		} else {
 			res.AmountFormatted = amountFormatted
 		}
+		res.WorthAmount = amountX.amountOfAsset
```

Comment:
Please add tests for this value to TestBuildPaymentLocal.  

I also think that if we are going to return WorthAmount, we should return WorthCurrency too.

Also, what is AmountFormatted that you added in the other PR used for?

Reply:
That was just to do e.g. `.5` => `0.5`. I put a comment in the go side but not in the avdl, I'll put it there too.